documentation/asciidoc/computers/raspberry-pi/bootflow-2711.adoc

```diff
@@ -70,9 +70,13 @@ Please see the xref:raspberry-pi.adoc#raspberry-pi-4-bootloader-configuration[bo
  ** else if boot-mode == `RPIBOOT` then
   *** Attempt to load firmware using USB device mode from the USB OTG port - see https://github.com/raspberrypi/usbboot[usbboot]. There is no timeout for `RPIBOOT` mode.
 
+==== Differences on Raspberry Pi5
+* The power button is used to wakeup from PMIC `STANDBY` or HALT instead of `GPIO 3`
+* Instead of loading `start.elf` the firmware loads the Linux kernel. Effectively, the bootloader has an embedded version of `start.elf`
```

Comment:
Small suggestion/nitpick.
```suggestion
==== Differences on Raspberry Pi 5
* The power button is used to wakeup from PMIC `STANDBY` or HALT instead of `GPIO 3`.
* Instead of loading `start.elf` the firmware loads the Linux kernel. Effectively, the bootloader has an embedded version of `start.elf`.
```
* use "Pi 5" instead of "Pi5"
* consistently format bullet points as sentences

Reply:
Fixed Pi5 rather than Pi 5 in 571e65e739d19d3157d87e8fc35e0777c0085ec1